### PR TITLE
DOCS-421 update Aerospike-configuration-mapping.md

### PR DESCRIPTION
**Pull request by ebarcott-aerospike**

### CODE REVIEW

File: docs/Aerospike-configuration-mapping.md [hunk 1]
```diff
@@ -6,19 +6,17 @@ description: Aerospike Configuration Mapping
 
 ## Mapping Between YAML and Aerospike Configuration
 
-Kubernetes uses [YAML](https://YAML.org/) to express its configuration whereas the Aerospike DB uses [it's own format for configuration](https://docs.aerospike.com/docs/configure/index.md) which it stores in `aerospike.conf`.
+Kubernetes uses YAML configuration files whereas the Aerospike DB uses its own format for configuration. The Aerospike DB configurations are stored in `aerospike.conf`.
```
Comment (arrowplum) [inline docs/Aerospike-configuration-mapping.md:9]: Why are we not linking to the config docs ( and schemas)?

Comment (ebarcott-aerospike) [reply docs/Aerospike-configuration-mapping.md:9]: The current thinking is we should not link out to anything at docs.aerospike.com until after the content migration. Post-migration, we'll go back in and add in a ton of links back and forth. If there are any links which are crucial to include now, please flag them like this and I'll put them back in (after tracking down a non-broken version of the link).

File: docs/Aerospike-configuration-mapping.md [hunk 2]
```diff
@@ -36,7 +34,7 @@ memory-size 4G
 
 ### Storage Sizes
 
-Memory, file, and devices sizes in the YAML format are integers and need to be specified as number of bytes. In `aerospike.conf` one may optionally provide a unit as a string.
+Memory, file, and device sizes in the YAML format are integers and need to be specified as number of bytes. In `aerospike.conf` one may optionally provide a unit as a string.
```
Comment (carlyorke99) [inline docs/Aerospike-configuration-mapping.md:37]:
```suggestion
Memory, file, and device sizes in the YAML format are integers and need to be specified as number of bytes. In `aerospike.conf` you may optionally provide a unit as a string.
```

**Review by carlyorke99 — APPROVED** [review]
LGTM

**Review by arrowplum — APPROVED** [review]

**Pull request merged**